cmd/snap-bootstrap/partition/sfdisk.go

```diff
@@ -270,30 +314,52 @@ func deviceLayoutFromPartitionTable(ptable sfdiskPartitionTable) (*DeviceLayout,
 		}
 		bd := info.BlockDevices[0]
 
+		vsType, err := fromSfdiskPartitionType(p.Type, ptable.Label)
+		if err != nil {
+			return nil, fmt.Errorf("cannot convert sfdisk partition type %q: %v", p.Type, err)
+		}
+
 		structure[i] = gadget.VolumeStructure{
 			Name:       p.Name,
 			Size:       gadget.Size(p.Size) * sectorSize,
 			Label:      bd.Label,
-			Type:       p.Type,
+			Type:       vsType,
 			Filesystem: bd.FSType,
 		}
+
 		ds[i] = DeviceStructure{
 			LaidOutStructure: gadget.LaidOutStructure{
 				VolumeStructure: &structure[i],
 				StartOffset:     gadget.Size(p.Start) * sectorSize,
 				Index:           i + 1,
 			},
 			Node:    p.Node,
-			Created: p.isCreated(),
+			Created: isCreatedDuringInstall(&p, &bd, ptable.Label),
```

Comment:
I missed that we had again a Created attr,  can we rename it to CreatedByBoostrap or CreatedDuringInstall?  all partitions are created somehow